src/js/controllers/changelly.js

```diff
@@ -1,176 +1,190 @@
-      walletService.getStatus(wallet, {}, function(err, status) {
-        if (err) {
-          $log.error(err);
-          return;
-        }
-        wallet.status = status;
-        updateTxps();
-      });
-    };
-
-    $scope.setAddress = function(newAddr) {
-      $scope.addr = null;
-      if (!$scope.wallet || $scope.generatingAddress || !$scope.wallet.isComplete()) return;
-      $scope.generatingAddress = true;
-      walletService.getAddress($scope.wallet, newAddr, function(err, addr) {
-        $scope.generatingAddress = false;
-
-        if (err) {
-          //Error is already formated
-          popupService.showAlert(err);
-        }
-
-        $scope.addr = addr;
-        $timeout(function() {
-          $scope.$apply();
-        }, 10);
-      });
-    };
-
-    $scope.openBackupNeededModal = function() {
-      $ionicModal.fromTemplateUrl('views/includes/backupNeededPopup.html', {
-        scope: $scope,
-        backdropClickToClose: false,
-        hardwareBackButtonClose: false
-      }).then(function(modal) {
-        $scope.BackupNeededModal = modal;
-        $scope.BackupNeededModal.show();
-      });
-    };
-
-    $scope.close = function() {
-      $scope.BackupNeededModal.hide();
-      $scope.BackupNeededModal.remove();
-    };
-
-    $scope.doBackup = function() {
-      $scope.close();
-      $scope.goToBackupFlow();
-    };
-
-    $scope.goToBackupFlow = function() {
-      $state.go('tabs.receive.backupWarning', {
-        from: 'tabs.receive',
-        walletId: $scope.wallet.credentials.walletId
-      });
-    };
-
-    $scope.shouldShowReceiveAddressFromHardware = function() {
-      var wallet = $scope.wallet;
-      if (wallet.isPrivKeyExternal() && wallet.credentials.hwInfo) {
-        return (wallet.credentials.hwInfo.name == walletService.externalSource.intelTEE.id);
-      } else {
-        return false;
-      }
-    };
-
-    $scope.showReceiveAddressFromHardware = function() {
-      var wallet = $scope.wallet;
-      if (wallet.isPrivKeyExternal() && wallet.credentials.hwInfo) {
-        walletService.showReceiveAddressFromHardware(wallet, $scope.addr, function() {});
-      }
-    };
-
-    var checkSelectedWallet = function(wallet, wallets) {
-      if (!wallet) return wallets[0];
-      var w = lodash.find(wallets, function(w) {
-        return w.id == wallet.id;
-      });
-      if (!w) return wallets[0];
-      return wallet;
-    }
-
-
-    $scope.openExternalLink = function(url) {
-      var optIn = true;
-      var title = null;
-      var message = gettextCatalog.getString('You will now be redirected to Changelly.com to complete your purchase');
-      var okText = gettextCatalog.getString('Okay');
-      var cancelText = gettextCatalog.getString('Go Back');
-      externalLinkService.open(url, optIn, title, message, okText, cancelText);
-    };
-
-    $scope.onWalletSelect = function(wallet) {
-      $scope.wallet = wallet;
-      $scope.setAddress();
-    };
-  });
+'use strict';
```

Comment:
What exactly changed in this file? (apart from what I assume are line ending changes)